Sources/Hummingbird/Middleware/FileMiddleware.swift

```diff
@@ -69,13 +70,16 @@ where Provider.FileAttributes: FileMiddlewareFileAttributes {
         threadPool: NIOThreadPool = NIOThreadPool.singleton,
         logger: Logger = Logger(label: "FileMiddleware")
     ) where Provider == LocalFileSystem {
-        self.cacheControl = cacheControl
-        self.searchForIndexHtml = searchForIndexHtml
-        self.urlBasePath = urlBasePath.map { String($0.dropSuffix("/")) }
-        self.fileProvider = LocalFileSystem(
-            rootFolder: rootFolder,
-            threadPool: threadPool,
-            logger: logger
+        self.init(
+            fileProvider: LocalFileSystem(
+                rootFolder: rootFolder,
+                threadPool: threadPool,
+                logger: logger
+            ),
+            urlBasePath: urlBasePath,
+            cacheControl: cacheControl,
+            searchForIndexHtml: searchForIndexHtml,
+            additionalMediaTypeExtensions: [:]
```

Comment:
mediaTypeFileExtensionMap?

Reply:
will do!

Reply:
Of should it be the other way fileExtensionMediaTypeMap? What do you think

Reply:
I think they are about equal in terms of clarity, but the first is more concise.

Reply:
> I think they are about equal in terms of clarity, but the first is more concise.

I just realized they are the same character count, but the second *looks* longer lol

Reply:
Just pushed - I also changed 
`public func withAdditionalMediaType(_ mediaType: MediaType, forFileExtension fileExtension: String) -> FileMiddleware`
to 
`public func withAdditionalMediaType(_ mediaType: MediaType, mappedToFileExtension fileExtension: String) -> FileMiddleware`
to match.